hc/src/help2.src

```diff
@@ -119,6 +119,15 @@ SUMMARY OF FRACTAL TYPES
     Three parameters: Filter Cycles, Seed Population, and Beta.
 
 ~OnlineFF
+{=HT_BURNINGSHIP burningship}
```

Comment:
`{=HT_BURNING_SHIP burning-ship}`

Reply:
The best I could do was "HT_BURNING_SHIP burningship" as changing burningship to burning-ship caused build errors.